modules/ROOT/pages/procedures.adoc

```diff
@@ -0,0 +1,2191 @@
+// * <<#_metrics, Metrics>>
+// * <<#_schema_and_metadata,Schema and metadata>>
+// * <<#_statistics_and_query_planning,Statistics and query planning>>
+// * <<#_transaction_management,Transaction management>>
+
+
+It also lists current xref:procedures.adoc#deprecated-procedures[deprecated procedures] and the xref:procedures.adoc#removed-procedures[procedures removed in Neo4j 5], along with their replacements.
+
+The available procedures on a server depends on several factors:
+
+* Neo4j Enterprise Edition provides a larger set of procedures than Neo4j Community Edition.
+* Neo4j's link:{neo4j-docs-base-uri}/apoc/{page-version}/[APOC Core library] and link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/genai-integrations/[GenAI plugin] are installed by default on link:{neo4j-docs-base-uri}/aura/[Aura] instances, but have to be installed separately on on-prem servers.
+* Cluster members have procedures that are not available in standalone mode.
+
+To check which procedures are available in your Neo4j DBMS, use the Cypher command link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/clauses/listing-procedures[`SHOW PROCEDURES`]:
+
+.List available procedures with default output columns
+[source, cypher]
+----
+SHOW PROCEDURES
+----
+
+.List available procedures with full output columns
+[source, cypher]
+----
+SHOW PROCEDURES YIELD *
+----
+
+
+[NOTE]
+====
+Some procedures can only be run by users with `Admin` privileges.
+Specifically, either the `EXECUTE ADMIN PROCEDURES` privilege or both the `EXECUTE PROCEDURES` and `EXECUTE BOOSTED PROCEDURES` privileges.
+These procedures are labeled with label:admin-only[].
+
+For more information, see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-execute[the `EXECUTE` privileges section].
+====
+
+== Authentication and authorization
+
+For more information, see xref:authentication-authorization/index.adoc[Authentication and authorization].
+
+[role=label--enterprise-edition label--admin-only]
+[[procedure_dbms_security_clearauthcache]]
+=== dbms.security.clearAuthCache()
+
+
+.Details
+|===
+| *Syntax* 3+m| dbms.security.clearAuthCache()
+| *Description* 3+a| Clears authentication and authorization cache.
+| *Mode* 3+| DBMS
+|===
+
+
+[[procedure_dbms_showcurrentuser]]
+=== dbms.showCurrentUser()
+
+
+.Details
+|===
+| *Syntax* 3+m| dbms.showCurrentUser() :: (username, roles, flags)
+| *Description* 3+a| Show the current user.
+.4+| *Return arguments* | *Name* | *Type* | *Description*
+| `username` | `STRING` | The name of the current user.
+| `roles` | `LIST<STRING>` | The roles assigned to the current user.
+| `flags` | `LIST<STRING>` | The flags set on the current user.
+| *Mode* 3+| DBMS
+|===
+
+
+== Background job management
+
+For more information, see xref:monitoring/background-jobs.adoc[].
+
+[[procedure_dbms_scheduler_failedjobs]]
+[role=label--enterprise-edition label--admin-only]
+=== dbms.scheduler.failedJobs()
```

Comment:
**Consider** using the Enterprise tag consistently. In this section each procedure is marked. In other sections, only the header is marked.